plugin.yml

```diff
@@ -1,5 +1,5 @@
-name: docker-ecr-cache
-description: Plugin for building and caching images to AWS ECR
+name: docker-ecr-publish
+description: Plugin for building, tagging, caching and publishing images to AWS ECR
```

Comment:
should this plugin accept a parameter for `tag`...maybe some other users would like to have a custom tagging strategy

Reply:
I think if we were going deep into making this good for other consumers we'd be consolidating some of these plugins too. I was thinking this would, at least for now just be something that serves our needs.

Reply:
should we consolidate now then before we start using them ourselves ?